docker/create-env.sh

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env bash
+
+# Copy the .dist environment files and wait them modifications.
+# Merge them to .env file for docker-compose.
+
+# Usage:
+# `./create-env.sh`
+# `./create-env.sh -q  # Quite mode`
+# `./create-env.sh --quite  # Quite mode`
+
+set -e
+
+QUITE=false
+for arg in "$@"
+do
+    if [[ $arg = "--quite" || $arg = "-q" ]]
+    then
+        QUITE=true
+    fi
+done
+
+
+function join_by { local IFS="$1"; shift; echo "$*"; }
+
+function create_env_files {
+    new_files=()
+    for file in env_files/*.dist
+    do
+        new_file=${file%.dist}
+        new_files+=( "$new_file" )
+
+        if [[ ! -f "$new_file" ]]
+        then
+            cp $file $new_file
+        fi
+    done
+
+    if ! $QUITE
+    then
+        file_names=$(join_by , ${new_files[@]})
+        read -p \
+"Go to this files and fill them with your own values.
```

Comment:
Looks too verbose. Maybe it will be better to say:
```
Configure environment files with values you need: $file_names.
Are you ready to merge them into a .env file?
```

Reply:
@artemiy312 , 
- `configure files with values` -> `fill files with values`
- proger: what values? Will someone give them for me? They lay in another part of system? -> `your own values`
- it's hard to parse file_names list from text, when they are on the same string

> Are you ready to merge them into a .env file?
- *bad:* proger: merge who? Files or values? ..., oh, it's the same thing
- *good:* `merge your values to .env file` -> proger: ok

> Hope you finished to configure files from list below.

this string we can remove.

So, i propose this:
```
Go to this files and fill them with your own values.
Files list: $file_names.
Ready to merge new config values to .env file? [y/n]:
```